bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -69,26 +78,88 @@
 				<description>Interval the car is polled in minutes.</description>
 				<default>10</default>
 			</parameter>
+			<parameter name="updateDelay" type="integer" unit="s" min="5" max="120">
+				<label>Update Delay</label>
+				<description>How long to wait for commands to reach car and update to server in seconds.</description>
+				<default>30</default>
+			</parameter>
 		</config-description>
 	</thing-type>
 
-	<!-- Sample Channel Type -->
-	<channel-type id="hvacstatus">
-		<item-type>Switch</item-type>
-		<label>HVAC Status</label>
+	<channel-type id="batteryavailableenergy">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Energy Available</label>
+		<state pattern="%.1f kWh" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="plugstatus">
+		<item-type>String</item-type>
+		<label>Plug Status</label>
+		<description>Status of charging plug.</description>
 		<state readOnly="true"></state>
 	</channel-type>
-	<channel-type id="image">
+	<channel-type id="chargingstatus">
 		<item-type>String</item-type>
-		<label>Image URL</label>
-		<description>Image URL of MyRenault</description>
+		<label>Charging Status</label>
 		<state readOnly="true"></state>
 	</channel-type>
+	<channel-type id="chargingmode">
+		<item-type>String</item-type>
+		<label>Charging Mode</label>
+	</channel-type>
```

Comment:
Please add state options to map your possible values to a proper label for UI, a label that could even be translated later in different languages.

Reply:
Done now with enums will run translation later once code stable.